packages/javascript-sdk/src/config/index.ts

```diff
@@ -60,6 +67,38 @@ abstract class Config {
     this.options = { ...setDefaults(options) };
   }
 
+  /**
+   * @method setAsync - Asynchronously calls the WellKnown endpoint to collect the APIs for OAuth
+   * @param {AsyncConfigOptions} options - config options with wellknown endpoint URL
+   * @returns {Promise<void>} - Returns a success or failure message object
+   */
+  public static async setAsync(options: AsyncConfigOptions): Promise<void> {
+    if (!options.serverConfig.wellknown) {
+      throw new Error(
```

Comment:
I suppose error throwing is our normal way of handling this, but is it worse if we instead just called `Config.set` ? I suppose it is because the developer may experience a behavior they aren't expecting, but also they have to deal with an error in this case. Just brain soup coming out in a comment.

Reply:
I agree. I'm not a fan of throwing errors either, but it's a convention throughout this project. I don't really want to introduce a new pattern at this moment.